src/models/DefaultConfiguration.js

```diff
@@ -51,6 +54,25 @@ class DefaultConfiguration {
      * @default []
      */
     this.tags = props.tags || [];
+    /**
+     Parameters for the ELK automatic layout system.
+     @see https://eclipse.dev/elk/reference/options.html
+     @type {object}
+     */
+    this.elkParams = {
```

Comment:
Missing test constructor in DefaultConfiguration.spec.js

Reply:
done